model2vec/model.py

```diff
@@ -66,7 +66,8 @@ def __init__(
             self._can_encode_fast = False
 
         if normalize is not None:
-            self.normalize = normalize
+            self._normalize = normalize
+            self.config["normalize"] = normalize
         else:
             self.normalize = self.config.get("normalize", False)
 
```

Comment:
Perhaps we can add `get_sentence_embedding_dimension()` like sentence transformers?

Reply:
this is already in here, it's called `dim`. It's a property, not a function.